### PR TITLE
Feature: Update account page design

### PR DESCRIPTION
**Pull request by Bilelkihal**
### Problem
The old interface has bad styling and does not provide a good user experience.

### What I did
- Fix hierarchy design problems and use modern styling.
- Add a button link to upload ontology directly from the user's account page (in case he didn't upload any ontology yet).
- Add a button to copy the API key.
- Add a link to the API documentation.

### Screenshots
**Old**

image

**New**

empty
Account

### CODE REVIEW

File: app/views/users/edit.html.haml [hunk 1]
```diff
@@ -1,145 +1,44 @@
...
-    width: auto;
-  }
-  table.form td {
-    width: 500px;
-  }
-
-:javascript
-  /* refer to input fields by name for rules */
-  /* refer to input fields by id for equalTo */
-  jQuery(document).ready(function() {
```
Comment (syphax-bouazzouni) [inline app/views/users/edit.html.haml:15]: you removed all this JS code without replacing it?

File: app/views/users/show.html.haml [hunk 2]
```diff
@@ -1,135 +1,116 @@
...
+    %h3.account-page-title My account
+    %hr#account-page-title-line/
+    .account-page-sub-container
+      .account-page-first-row
+        .account-page-card
+          .account-page-personal-informations-title-bar
+            %h4.account-page-card-title Personal information
+            %a.account-page-rounded-button{:href => edit_user_path(url_encode(@user.username))}
+              / SVG: Edit icon
+              %svg{fill: "none", height: "18", viewbox: "0 0 18 18", width: "18", xmlns: "http://www.w3.org/2000/svg"}
```
Comment (syphax-bouazzouni) [inline app/views/users/show.html.haml:12]: extract SVG in files, it make it hard to read the file

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

File: app/views/users/edit.html.haml (hunk 1 above)
Comment (Bilelkihal) [reply app/views/users/edit.html.haml:15]: This file is still in progress

**Comment by syphax-bouazzouni** [comment]
Some bugs found

## subscribe button to aligned

image


### mailing list incorrect subject and body

image

### Orcid validation 

image

**Comment by Bilelkihal** [comment]
Okay 👌

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

**Comment by Bilelkihal** [comment]
I've fixed all the problems except for the subscription mail, can you please give me the subject & body to put, or should I think about writing it by myself?

**Review by syphax-bouazzouni — APPROVED** [review]

**Comment by syphax-bouazzouni** [comment]
Merged, @Bilelkihal  add the correct Subscription mail subject and body in another PR.

**Pull request merged**